### DISCUSSION TITLE
lmplot and applying condition on either axis.

### DISCUSSION THREAD

**Discussion by mahnoor-byte** [AammarTufail/python_ka_chilla2023#66] [Q&A]
I was not able to add condition in one line but the following code worked for me. Is there any other way as well to do this?

```
p = sns.lmplot(data=df, x="age", y="fare", hue="who")
p.set(ylim=(0,300))
```

**Comment by MuhammadHammad4127** [comment]
Code: sns.lmplot(y='age', x='fare', hue='survived' ,data=df).set(xlim=(0,250))
